seed/tests/test_analyses.py

```diff
@@ -0,0 +1,176 @@
+# !/usr/bin/env python
+# encoding: utf-8
+"""
+:copyright (c) 2014 - 2020, The Regents of the University of California, through Lawrence Berkeley National Laboratory (subject to receipt of any required approvals from the U.S. Department of Energy) and contributors. All rights reserved.  # NOQA
+:author
+"""
+from django.test import TestCase
+import json
+
+from seed.landing.models import SEEDUser as User
+from seed.models import (
+    Analysis,
+    AnalysisPropertyView
+)
+from seed.test_helpers.fake import (
+    FakeCycleFactory,
+    FakePropertyFactory,
+    FakePropertyStateFactory
+)
+from seed.utils.organizations import create_organization
+
+
+class TestAnalyses(TestCase):
+
+    def setUp(self):
+        user_details = {
+            'username': 'test_user@demo.com',
+            'password': 'test_pass',
+            'email': 'test_user@demo.com',
+            'first_name': 'Test',
+            'last_name': 'User',
+        }
+        self.user = User.objects.create_user(**user_details)
+        self.org, self.org_user, _ = create_organization(self.user)
+        self.org_b, self.org_user, _ = create_organization(self.user)
+        self.client.login(**user_details)
+
+        cycle_factory = FakeCycleFactory(organization=self.org, user=self.user)
+        cycle_a = cycle_factory.get_cycle(name="Cycle A")
+        cycle_b = cycle_factory.get_cycle(name="Cycle B")
+
+        property_factory = FakePropertyFactory(organization=self.org)
+        self.property_a = property_factory.get_property()
+        property_b = property_factory.get_property()
+
+        property_state_factory = FakePropertyStateFactory(organization=self.org)
+        property_state_a = property_state_factory.get_property_state()
+        property_state_b = property_state_factory.get_property_state()
+        property_state_c = property_state_factory.get_property_state()
+        property_state_d = property_state_factory.get_property_state()
+
+        # create an analysis with two property views, each with the same property but a different cycle
+        self.analysis_a = Analysis.objects.create(
+            name='test a',
+            service=1,
+            status=10,
+            user=self.user,
+            organization=self.org
+        )
+        AnalysisPropertyView.objects.create(
+            analysis=self.analysis_a,
+            property=self.property_a,
+            cycle=cycle_a,
+            property_state=property_state_a
+        )
+        AnalysisPropertyView.objects.create(
+            analysis=self.analysis_a,
+            property=self.property_a,
+            cycle=cycle_b,
+            property_state=property_state_b
+        )
```

Comment:
@Ryoken Re: https://github.com/SEED-platform/seed/pull/2473#discussion_r534513955 , 
Could it be related to these two -Views associated to the same Property? That is a case we should accommodate for (This can happen during merging 🤮 , but let's not think about that yet!).